app/controllers/posts_controller.rb

```diff
@@ -5,6 +5,7 @@ class PostsController < ApplicationController
   def index
     @post = Post.new
     @posts = current_user.posts.all.order("created_at DESC")
+    @tag = @post.tags.new
```

Comment:
Hmm, are you trying to create `associated objects` when creating one `primary object`?